src/System.Linq/src/Var.cs

```diff
@@ -1,5 +1,4 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-internal static class var { } // Enforce explicit declaration of variable types.
-
 
+internal static class var { } // Enforce explicit declaration of variable types.
```

Comment:
Why the change to this file? (FWIW, I personally think we should get rid of this file, as our coding guidelines do allow for the use of var in some cases, and even if they didn't, eventually a Roslyn analyzer would be the preferred way to prevent its usage.  But if we were to get rid of it, we'd want to do it separately from test improvements.)